cpp/src/plasma/format/plasma.fbs

```diff
@@ -215,8 +215,9 @@ table PlasmaStatusRequest {
 }
 
 enum ObjectStatus:int {
+  Null = 0,
```

Comment:
This is now fixed in the latest master, if you rebase or merge the master into this branch it should work!